app/uk/gov/hmrc/apisubscriptionfields/controller/ApiSubscriptionFieldsController.scala

```diff
@@ -34,24 +34,28 @@ class ApiSubscriptionFieldsController @Inject() (service: SubscriptionFieldsServ
 
   import JsonFormatters._
 
-  private val NOT_FOUND_RESPONSE = NotFound(JsErrorResponse(SUBSCRIPTION_FIELDS_ID_NOT_FOUND, "Subscription Fields were not found"))
+  private def notFoundResponse(message: String) = {
+    NotFound(JsErrorResponse(NOT_FOUND_CODE, message))
+  }
+
+  private val ID_NOT_FOUND = "Id not found"
```

Comment:
This could be confusing, as in the endpoints you pass the 3 parameters (application id, api context, api version) separately, not aggregated together as a single parameter (id). 
Thus, I would suggest to replace it as:
```
Id (application id, api context, api version) was not found
```